src/components/Editor/CustomExtensions/CodeBlock/constants.js

```diff
@@ -0,0 +1,6 @@
+import { lowlight } from "lowlight";
+
+export const SORTED_LANGUAGE_LIST = lowlight
+  .listLanguages()
+  .concat("auto")
+  .sort();
```

Comment:
"auto" must appear on the top of the list. So please append it to the front after sorting the original list.